products/green_fast_track/models/product/_product_models.yml

```diff
@@ -79,7 +77,38 @@ models:
         tests: [not_null]
       - name: '"(E) Designations - HazMat"'
         data_type: string
+      - name: '"Federal Parks property"'
+        data_type: string
+        tests: [not_null]
+      - name: '"Federal Parks property Flag"'
+        data_type: string
+      - name: '"NYC Parks properties"'
+        data_type: string
+        tests: [not_null]
+      - name: '"NYC Parks properties Flag"'
+        data_type: string
+      - name: '"NYS Parks Property"'
+        data_type: string
+        tests: [not_null]
+      - name: '"NYS Parks Property Flag"'
+        data_type: string
+      - name: '"POPS"'
+        data_type: string
+        tests: [not_null]
+      - name: '"POPS Flag"'
+        data_type: string
+      - name: '"Publicly Owned Waterfront"'
+        data_type: string
+        tests: [not_null]
+      - name: '"Publicly Owned Waterfront Flag"'
+        data_type: string
+      - name: '"Waterfront Public Access Areas"'
+        data_type: string
+        tests: [not_null]
+      - name: '"Waterfront Public Access Areas Flag"'
+        data_type: string
       - name: geom
+        data_type: geometry(Geometry, 2263)
```

Comment:
Nice!!! Do you know whether it checks projection at a column or value level? I'm wondering if it pulls the crs value from geom columns table in Postgres...

Reply:
pretty sure it's the column level. I just added a snippet from the docs the PR description that seems to indicate it isn't value level

I imagine value-level contract enforcement would add a lot of time to building a model

Reply:
Got it. Do you know if it raises an error if the projection value is zero at the column level?

Reply:
in this case it would because the data type of `geometry(Geometry, 2263)` is expecting a projection of 2263